seqr/views/apis/individual_api_2_3_tests.py

```diff
@@ -223,7 +225,9 @@ def _is_expected_hpo_upload(self, response):
             'uploadedFileId': mock.ANY,
             'errors': [],
             'warnings': [
-                "The following HPO terms were not found in seqr's HPO data and will not be added: HP:0004322 (NA19675_1); HP:0100258 (NA19679)",
+                # The order of the individual list in the first warning message can change.
+                # "The following HPO terms were not found in seqr's HPO data and will not be added: HP:0004322 (NA19675_1); HP:0100258 (NA19679)",
+                mock.ANY,
```

Comment:
why not change the code to sort the HPO terms when generating the warning?

Reply:
Updated